sdk/provisioning/Azure.Provisioning/tests/Infrastructure/RoleAssignmentWithoutParameterInteractiveMode/main.bicep

```diff
@@ -15,18 +15,18 @@ resource storageAccount_RFwKam5MV 'Microsoft.Storage/storageAccounts@2022-09-01'
   }
 }
 
-resource blobService_cIUg9iT7j 'Microsoft.Storage/storageAccounts/blobServices@2022-09-01' = {
-  parent: storageAccount_RFwKam5MV
+resource blobService_A96Cuv27c 'Microsoft.Storage/storageAccounts/blobServices@2022-09-01' = {
+  parent: storageAccount_XJNeT6Eti
   name: 'default'
   properties: {
   }
 }
 
-resource roleAssignment_4Wo9FVPss 'Microsoft.Resources/roleAssignments@2022-04-01' = {
-  scope: storageAccount_RFwKam5MV
-  name: guid('storageAccount_RFwKam5MV', '00000000-0000-0000-0000-000000000000', 'ba92f5b4-2d11-453d-a403-e96b0029c9fe')
+resource roleAssignment_5PbYbRPAD 'Microsoft.Resources/roleAssignments@2022-04-01' = {
+  scope: storageAccount_XJNeT6Eti
+  name: guid('storageAccount_XJNeT6Eti', '00000000-0000-0000-0000-000000000000', 'ba92f5b4-2d11-453d-a403-e96b0029c9fe')
```

Comment:
this doesn't match the function they are using is there any reason we are deviating?https://github.com/dotnet/aspire/blob/main/src/Aspire.Hosting.Azure/Bicep/storage.bicep#L34